bio/seqkit/wrapper.py

```
@@ -2,19 +2,35 @@
 
 
 extra = snakemake.params.get("extra", "")
 log = snakemake.log_fmt_shell(stdout=False, stderr=True)
 
-# subcommands concat and common use multiple input files
+
+# Subcommands with multiple input files
 if snakemake.params.command in ["concat", "common", "stats"]:
     input = " ".join(snakemake.input)
+# Subcommands with a single input file (if more than one provided, concat'em all)
+elif snakemake.params.command in [
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)